xspf2m3u.py

```diff
@@ -158,7 +166,14 @@ def main():
                 s = ' - '.join('{}: {}'.format(k, v) for k, v in track.items())
                 print('# Warning: ' + s)
             else:
-                print(location)
+                 if target == "M3U":
+                    print(location)
+                 if target == "M3UEXT":
+                    duration = int(int(track['duration']) / 1000) #I am not sure about this
```

Comment:
Better use `int(track['duration'], 10) // 1000`.

What exactly are you not sure about? For me this looks fine.

Reply:
ok